### PR TITLE
Adding basic skeleton for testing controllers. (stole from melange)

### PR DESCRIPTION
**Pull request by hub-cap**
- Built out the functional tests (db sync)
- Built out the actual database controller test get call

### CODE REVIEW

File: reddwarf/tests/unit/test_database_service.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+from reddwarf import tests
+from reddwarf.common import config
+from reddwarf.common import wsgi
+from reddwarf.database import models
+from reddwarf.database import service
+from reddwarf.tests import unit
+
+
+LOG = logging.getLogger(__name__)
```
Comment (ed-) [inline reddwarf/tests/unit/test_database_service.py:28]: more of this everywhere please

**Comment by TimSimpsonR** [comment]
+1

**Comment by cp16net** [comment]
+1

**Comment by ed-** [comment]
+1

**Pull request merged**